tools/docker/images/builder/README.md

```diff
@@ -0,0 +1,9 @@
+# Docker builder image
+
+Included in this folder is a Dockerfile which builds prplMesh, and can be used to verify build by CI.
+
+## Usage
+
+Build the initial image (one time) - `sudo docker image build --tag $USER/prpl-ci-build .`
```

Comment:
Since the initial image is not user-specific, I would leave out the `$USER` part from the default tag. Also, I'd name it prplMesh, and leave out the `ci` bit (because it can also be used for normal builds). And finally, I think examples that are to be cut-and-pasted should not be inline.
```suggestion
Build the initial image (one time):
    sudo docker image build --tag prplmesh-build .
```
 (It's difficult to put code within code so I used 4 spaces instead of 3 backticks, but backticks are better...)